app/Resources/public/less/index.less

```diff
@@ -216,6 +216,19 @@ html[lang|="en"] .promo-small-landscape.privacy {
     font-size: 1.3em;
 }
 
+.promo-large-portrait.ndg {
+    background-color: #27AE60;
+    background-image: url("/images/camera.svg");
```

Comment:
Yes, this just adds a scrollbar if necessary. E.g. the ugly horizontal scrollbar is now hidden.